From 7a929e88beab1c33cfef5df81e3ba4dcb596a570 Mon Sep 17 00:00:00 2001
From: Guan Jing <guanjing6@huawei.com>
Date: Wed, 30 Nov 2022 18:23:26 +0800
Subject: [PATCH 082/500] [Huawei] sched/fair: Disables SIS_UTIL and enables
 SIS_PROP

Offering: HULK
hulk inclusion
category: feature
bugzilla: 188420

--------------------------------

When doing wakeups, attempt to limit superfluous scans of the LLC domain.
We disables SIS_UTIL and enables SIS_PROP to close search idle CPU based on
sum of util_avg.

Signed-off-by: Guan Jing <guanjing6@huawei.com>
---
 kernel/sched/features.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/sched/features.h b/kernel/sched/features.h
index 3f9129f6bc6b..67672b8d57ae 100644
--- a/kernel/sched/features.h
+++ b/kernel/sched/features.h
@@ -54,8 +54,8 @@ SCHED_FEAT(TTWU_QUEUE, true)
 /*
  * When doing wakeups, attempt to limit superfluous scans of the LLC domain.
  */
-SCHED_FEAT(SIS_PROP, false)
-SCHED_FEAT(SIS_UTIL, true)
+SCHED_FEAT(SIS_PROP, true)
+SCHED_FEAT(SIS_UTIL, false)
 
 #ifdef CONFIG_SCHED_STEAL
 /*
-- 
2.17.1

